src/source/Signaling/Client.c

```diff
@@ -1,13 +1,37 @@
 #define LOG_CLASS "SignalingClient"
 #include "../Include_i.h"
 
+STATUS validateSignalingClientRetryStrategy(PSignalingClientInfo pClientInfo) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PSignalingClientRetryStrategy pSignalingClientRetryStrategy;
+
+    CHK(pClientInfo != NULL, STATUS_NULL_ARG);
+
+    pSignalingClientRetryStrategy = &(pClientInfo->signalingClientRetryStrategy);
+
+    CHK(pSignalingClientRetryStrategy->retryStrategyType > KVS_RETRY_STRATEGY_DISABLED &&
```

Comment:
`pSignalingClientRetryStrategy->retryStrategyType > KVS_RETRY_STRATEGY_DISABLED` to return `STATUS_NULL_ARG` seems misleading. Maybe we can look into a more suitable status code? What is the behavior in PIC if this retry strategy is used? I am wondering if `STATUS_NOT_IMPLEMENTED` would be better.

Reply:
Agree. STATUS_NOT_IMPLEMENTED would be better. Fixing it. I did not find that earlier. If KVS_RETRY_STRATEGY_DISABLED is used, then no wait times will be injected on failures (current behavior).